v2.1/create-security-certificates-custom-ca.md

```diff
@@ -0,0 +1,117 @@
+
+<div class="filters filters-big clearfix">
+  <a href="create-security-certificates.html"><button style="width:28%" class="filter-button">Use cockroach cert</button>
+  <a href="create-security-certificates-openssl.html"><button style="width:28%" class="filter-button">Use Openssl</button></a>
+  <button style="width:28%" class="filter-button current">Use <strong>custom CA</strong></button></a>
+</div>
+
+This document discusses the following advanced use cases for using security certificates with CockroachDB:
+
+Approach | Use case description
+-------------|------------
+UI certificate and key | When you want to access the Admin UI for a secure cluster and avoid clicking through a warning message to get to the UI.
+Split-node certificate | When your organizational CA requires you to have separate certificates for when the node acts as a server and as a client.
+Split-CA certificates | When you have multiple CockroachDB clusters and need to restrict access to clients from accessing the other cluster
+
+## Accessing the Admin UI for a secure cluster
+
+On [accessing the Admin UI](admin-ui-access-and-navigate.html#access-the-admin-ui), your browser will consider the CockroachDB-created certificate invalid, so you’ll need to click through a warning message to get to the UI.
+
+For secure clusters, you can avoid getting the warning message by using a certificate issued by a public CA:
+
+1. Request a certificate from a public CA (for example, [Let's Encrypt](https://letsencrypt.org/)). The certificate must have the IP addresses and DNS names used to reach the Admin UI listed in the `Subject Alternative Names` field.
+2. Rename the certificate and key as `ui.crt` and `ui.key`.
+3. Add the `ui.crt` and `ui.key` to the [certificate directory](create-security-certificates.html#certificate-directory). `ui.key` must not have group or world permissions (maximum permissions are 0700, or rwx------). This check can be disabled by setting the environment variable `COCKROACH_SKIP_KEY_PERMISSION_CHECK=true`.
+4. For nodes that are already running, load the `ui.crt` certificate without restarting the node by issuing a `SIGHUP` signal to the cockroach process:
+   {% include copy-clipboard.html %}
+   ~~~ shell
+   pkill -SIGHUP -x cockroach
+   ~~~
+   The `SIGHUP` signal must be sent by the same user running the process (e.g., run with sudo if the cockroach process is running under user root).
+
+### Node key and certificates
+
+File name pattern | File usage
+-------------|------------
+`ca.crt`     | CA certificate
+`node.crt`   | Server certificate. <br><br> `node.crt` must have `CN=node` and the list of IP addresses and DNS names listed in `Subject Alternative Names` field. <br><br>Must be signed by `ca.crt`
+`node.key`   | Key for server certificate
+`ui.crt` | UI certificate. `ui.crt` must have the IP addresses and DNS names used to reach the Admin UI listed in `Subject Alternative Names`
```

Comment:
Consider bolding the ui.crt and ui.key lines since they are the unique part of this section (and doing the same for `client.node.*` and ca-client.crt below).